pkg/grafana/alertrule-handler.go

```diff
@@ -0,0 +1,203 @@
+package grafana
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"encoding/json"
+	"errors"
+
+	"github.com/grafana/grizzly/pkg/grizzly"
+	"github.com/grafana/tanka/pkg/kubernetes/manifest"
+
+	"github.com/grafana/grafana-openapi-client-go/client/provisioning"
+	"github.com/grafana/grafana-openapi-client-go/models"
+)
+
+// AlertRuleHandler is a Grizzly Handler for Grafana alertRules
+type AlertRuleHandler struct {
```

Comment:
Does a  AlerRuleHandler make sense? The API will just be updating a single rule inside an AlertRuleGroup right? So actually this makes things more complicated than they need be?

Reply:
I'm open to just having an `AlertRuleGroup` handler. It mostly comes down to ergonomics :shrug:

Reply:
I've pushed a PR now to reduce this to only use AlertRuleGroups